### ISSUE TITLE
Global variables are cached and haven't cleaned between two calls of sitespeed.run()

### ISSUE THREAD

**Issue by suratovvlad** [sitespeedio/sitespeed.io#1851]
Hi,

I'm trying to use sitespeed.io as npm library, and I've faced with strange issue - the first call of `sitespeed.run()` finishes successfully, but the others calls - crahes with error: `ERROR: Error: Multiple summary messages of type aggregateassets.summary and group total`.

[There is a simple example that uses sitespeed.io as library](https://github.com/suratovvlad/testing_sitespeedio_bug). It should be started as `npm start`.

[There is the output of program](https://gist.github.com/suratovvlad/ed401784a8f224b4604f19acfebee819)

I've tested it with version 5.6.5 and with version 6.0.3. The results are same - error.

Also, I've made some investigations and found an issue, solution for which could cause a problem. I guess there were the issue #1725 and the commit d4c7832bc78406e49ec8c7d1c2f8127fff23b686.

In this commit, mr. @tobli  have added the variable `const groupsPerSummaryType = {};`, and now this variable is cached somehow by js engine after first call of `sitespeed.run()` (I've check it under debugging, it wasn't empty on the second call). I suppose it could be cause by closure mechanism, but I'm not the expert in JS.

Please, could you check it?

**Comment by soulgalore** [comment]
@suratovvlad yep there's another reported issue, I'll add a reference later. There's multiple places that need to be fixed for this to work across the board, easiest, for now, is to create a new sitespeed.io object and we can take on the main problem later.

**Comment by johnnyxh** [comment]
Have there been any updates on this? I'm trying to do something similar using version 7.6.3 and getting the same results. Deleting the entry in the require.cache and re-importing it is also giving me the same result.

**Comment by soulgalore** [comment]
Nothing done there, the way to do it right now is to instantiate a new object.
